app/services/document_publishing_service.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'gds_api/publishing_api_v2'
+
+class DocumentPublishingService
+  def publish_draft(document)
+    publishing_api.put_content(document.content_id, payload(document))
+  end
+
+  def publish(document)
+    publishing_api.publish(document.content_id, "major")
+  end
+
+private
+
+  def payload(document)
+    {
+      base_path: "/test/government/foo",
```

Comment:
Shall we generate a uuid or something here for proof of concept? otherwise once you'll keep overriding same content / get blocked.

Reply:
As there's no way to see what the URL is at the moment, I don't think there's much point. We should still be able to publish a single document, though 😄 .

Reply:
Is that OK with you?

Reply:
d'oh yeah good point on not knowing what the URL is heh. Yeah sure.